.github/workflows/scripts/pr_workflow.py

```diff
@@ -297,6 +305,19 @@ def issue_comment_label_actions(
     def add_welcome_comment(self) -> None:
         self.pr.create_issue_comment(body=WELCOME_COMMENT)
 
+    def approve_pr(self) -> None:
+        self.pr.create_review(event="APPROVE")
+
+    def dismiss_pr_approval(self) -> None:
+        all_reviews = self.pr.get_reviews()
+        current_user = self.gh_client.get_user().login
```

Comment:
isn't this `self.user_login`?

Reply:
No, the approval through the action is left by the bot, so we need to dismiss the bot's review.
We could expand it to also remove for the user who has left the comment if an approval by them exists, but we run into the issue that we don't have a token for each user and we won't be able to dismiss it using the bot's token